doc/lsst.ap.verify/command-line-reference.rst

```diff
@@ -0,0 +1,139 @@
+.. _ap-verify-cmd:
```

Comment:
I think it'd be useful to add

```rst
.. program:: ap_verify.py
```

here.

This way the options documented below can be uniquely referenced as

```rst
:option:`ap_verify.py --dataset`
```

or

```rst
:option:`--dataset <ap_verify.py --dataset>`
```

for short

(http://www.sphinx-doc.org/en/stable/domains.html?highlight=option#directive-program)

Reply:
Ok. I was using https://raw.githubusercontent.com/lsst/pipe_base/master/doc/lsst.pipe.base/command-line-task-argument-reference.rst as a template, which doesn't have that.

Reply:
Yeah, maybe I could add `program`, but in that case there was no specific script being reference, so it seemed safe to have the flags default to pointing to that command-line task page rather than being namespaced.